include/tvm/tir/schedule/schedule.h

```diff
@@ -364,6 +364,17 @@ class ScheduleNode : public runtime::Object {
    */
   virtual void ReverseComputeInline(const BlockRV& block) = 0;
   /******** Schedule: Reduction ********/
+  /*!
+   * \brief Decompose a reduction block into init block and update block, where the newly generated
+     init block will be before the specified loop.
+     1) The block is a reduction block.
+     2) The loop is the ancestor of the block.
+     3) The loop is not lower than all the loops related to reduce block var.
+   * \param block_rv The reduction block to be decomposed
+   * \param loop_rv The position where init block is inserted
+   * \return The init block
+   */
+  virtual BlockRV DecomposeReduction(const BlockRV& block_rv, const LoopRV& loop_rv) = 0;
```

Comment:
Did we discontinue our support for the case where `loop_rv == None`?

Reply:
Yes